nominations.jq

```diff
@@ -0,0 +1,11 @@
+if length >= 100 then
```

Comment:
If we ever run into this issue, the `gh` tool from github should be able to handle pagination.